ci/jenkins/test.sh

```diff
@@ -828,181 +812,7 @@ function clean_tmp() {
     find ${WORKDIR} -name "support-bundles*" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
 }
 
-# redeploy_k8s_if_ip_mode_changes redeploys K8s cluster when flexible-ipam is set and existing cluster ip-mode doesn't match input ip-mode
-function redeploy_k8s_if_ip_mode_changes() {
-    echo "===== Read K8s Node info ====="
-    WORKER_HOSTNAMES=()
-    HOSTNAMES=()
-    WORKER_IPV4S=()
-    IPV4S=()
-    WORKER_IPV6S=()
-    IPV6S=()
-    NODE_HOSTNAMES=`cat ${WORKDIR}/nodes | awk '{print $1}'`
-    while read HOSTNAME; do
-        if [ ${CONTROL_PLANE_HOSTNAME} ]; then
-            WORKER_HOSTNAMES+=("${HOSTNAME}")
-        else
-            CONTROL_PLANE_HOSTNAME=${HOSTNAME}
-        fi
-        HOSTNAMES+=("${HOSTNAME}")
-    done <<< ${NODE_HOSTNAMES}
-    NODE_IPV4S=`cat ${WORKDIR}/nodes | awk '{print $2}'`
-    while read IPV4; do
-        if [ ${CONTROL_PLANE_IPV4} ]; then
-            WORKER_IPV4S+=("${IPV4}")
-        else
-            CONTROL_PLANE_IPV4=${IPV4}
-        fi
-        IPV4S+=("${IPV4}")
-    done <<< ${NODE_IPV4S}
-    NODE_IPV6S=`cat ${WORKDIR}/nodes | awk '{print $3}'`
-    while read IPV6; do
-        if [ ${CONTROL_PLANE_IPV6} ]; then
-            WORKER_IPV6S+=("${IPV6}")
-        else
-            CONTROL_PLANE_IPV6=${IPV6}
-        fi
-        IPV6S+=("${IPV6}")
-    done <<< ${NODE_IPV6S}
-
-    echo "===== Check K8s cluster status ====="
-    INITIAL_VALUE=10
-    HAS_IPV4=${INITIAL_VALUE}
-    HAS_IPV6=${INITIAL_VALUE}
-    POD_CIDRS=($( (kubectl get node ${CONTROL_PLANE_HOSTNAME} -o json | jq -r '.spec.podCIDRs | @sh') | tr -d \'\")) || true
-    echo "POD_CIDRS=${POD_CIDRS[*]}"
-    for POD_CIDR in "${POD_CIDRS[@]}"; do
-        if [[ $POD_CIDR =~ .*:.* ]]
-        then
-            (( HAS_IPV6++ ))
-        else
-            (( HAS_IPV4++ ))
-        fi
-    done
-    if [[ ${IP_MODE} == "ipv4" ]]; then
-        (( HAS_IPV4-- ))
-    elif [[ ${IP_MODE} == "ipv6" ]]; then
-        (( HAS_IPV6-- ))
-    else
-        (( HAS_IPV4-- ))
-        (( HAS_IPV6-- ))
-    fi
-    if [ ${HAS_IPV4} -eq ${INITIAL_VALUE} ] && [ ${HAS_IPV6} -eq ${INITIAL_VALUE} ]; then
-      return 0
-    fi
-
-    echo "===== Reset K8s cluster Nodes ====="
-    for IPV4 in "${IPV4S[@]}"; do
-        ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/id_rsa" -n ubuntu@${IPV4} "sudo kubeadm reset -f; sudo netplan apply; sudo ip link set br-int down || true"
-    done
-
-    echo "===== Redeploy K8s utils ====="
-    sudo apt update < /dev/null
-    sudo apt remove -y kubectl < /dev/null
-    sudo apt install -y kubectl=${K8S_VERSION} < /dev/null
```

Comment:
Is `K8S_VERSION` still being used in the script? You can refer to the previous IPAM job commit to check if there is any other redundant code that can be removed: https://github.com/antrea-io/antrea/commit/ce3059e5ed21b0a8086c7a310e7d494a2ac14b67

Reply:
yes done.